src/test/ui/macros/bad_hello.stderr

```diff
@@ -2,7 +2,7 @@ error: expected a literal
   --> $DIR/bad_hello.rs:12:14
    |
 LL |     println!(3 + 4); //~ ERROR expected a literal
-   |              ^^^^^
+   |              ^^^^^ help: consider changing this to: `"{}", 3 + 4`
```

Comment:
I think this suggestion as-is could be a little confusing: selecting the entire expression and suggesting it be replaced with `println!("{}", 3 + 4)` would give more helpful context.